src/components/Lightboxes/AddToBasketInterstitial.vue

```diff
@@ -93,50 +96,70 @@ import cookieStore from '@/util/cookieStore';
 import basketAddInterstitial from '@/graphql/query/basketAddInterstitialClient.graphql';
 import basketAddInterstitialData from '@/graphql/query/basketAddInterstitialData.graphql';
 import updateAddToBasketInterstitial from '@/graphql/mutation/updateAddToBasketInterstitial.graphql';
-import KvLightbox from '@/components/Kv/KvLightbox';
 import KvButton from '@/components/Kv/KvButton';
+import KvCheckbox from '@/components/Kv/KvCheckbox';
+import KvLightbox from '@/components/Kv/KvLightbox';
+import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
 import LoanReservation from '@/components/Checkout/LoanReservation';
 import LYML from '@/components/LoansYouMightLike/lymlContainer';
-import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
 
 export default {
 	components: {
-		KvLightbox,
 		KvButton,
+		KvCheckbox,
+		KvLightbox,
+		KvLoadingSpinner,
 		LoanReservation,
 		LYML,
-		KvLoadingSpinner,
 	},
 	inject: ['apollo'],
 	data() {
 		return {
-			showInterstitial: false,
 			basketInterstitialState: {},
 			loan: {},
 			loans: () => [],
 			loanCount: 0,
 			loanTotals: '0.00',
 			loading: true,
+			showInterstitial: false,
+			userPrefHideInterstitial: false,
 		};
 	},
 	computed: {
 		basketInterstitialActive() {
 			return this.basketInterstitialState.active || false;
 		}
 	},
+	watch: {
+		userPrefHideInterstitial() {
+			try {
+				localStorage.setItem('userPrefHideInterstitial', this.userPrefHideInterstitial);
```

Comment:
Use the `store2` package rather than using local storage directly. When you do that, there's no need for the try/catch

Reply:
Ooh, it actually stores the value as a Boolean instead of String. 😲